src/tags/templates/tags/snippets/related-xp-item.html

```diff
@@ -20,7 +20,7 @@
             <p></p>
             <ul>
                 {% for badge in badges %}
-                    <li><a href="{{ badge.get_absolute_url }}">{{ badge }}</a></li>
+                    <li><a href="{{ badge.get_absolute_url }}">{{ badge }} ({{ badge.xp }} XP)</a></li>
```

Comment:
Similar to above, are these badges or assertions? Does this take into account multiple awardings of the same badge?

I guess same questions then for the Profile view of XP by tags

Reply:
" are these badges or assertions?"
these are badges

"Does this take into account multiple awarding's of the same badge?"
no since it was never a feature to be implemented in (#1125)